packages/manager/apps/dedicated/client/app/ip/components/list/list.html

```diff
@@ -0,0 +1,416 @@
+<div class="table-responsive mt-3 module-ip-table-responsive">
+    <table class="table table-hover" id="module-ip-table">
```

Comment:
**question:** Now that we have a separate tabs for all the types of IPs, can we use `oui-datagrid` with expandable rows instead?
Ref: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-datagrid-webcomponent--expandable-rows